### PR TITLE
chore(macros): update zh-CN strings

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
uniform some descriptions.
update the `learn` macros

### CODE REVIEW

**Comment by awxiaoxian2020** [comment]
cc: @yin1999

File: kumascript/macros/LearnSidebar.ejs [hunk 1]
```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
...
-      'Client-side_web_APIs': '客户端网页 API',
-        'Client-side_web_APIs_Introduction': '网页 API 介绍',
-        'Client-side_web_APIs_Manipulating_documents': '操纵文档',
+        'Assessment_Adding_features_to_our_bouncing_balls_demo' : '测验：为弹跳球展示新增功能',
+      'Asynchronous_JavaScript': '异步 JavaScript',
+        'Asynchronous_JavaScript_overview':'异步 JavaScript 概述',
+        'Introducing_asynchronous_JavaScript': '异步 JavaScript 简介',
+        'How_to_use_promises': '如何使用 promise',
+        'Implementing_a_promise-based_API': '实现基于 promise 的 API',
+        'Introducing_workers': 'workers 简介',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1170]:
```suggestion
        'Introducing_workers': 'worker 简介',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 2]
```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
...
       'React': 'React',
-        'Getting_started_with_React': 'Getting started with React',
-        'Beginning_our_React_todo_list': 'Beginning our React todo list',
-        'Componentizing_our_React_app': 'Componentizing our React app',
-        'React_interactivity:_Events_and_state': 'React interactivity: Events and state',
-        'React_interactivity:_Editing_filtering_conditional_rendering': 'React interactivity: Editing, filtering, conditional rendering',
-        'Accessibility_in_React': 'Accessibility in React',
-        'React_resources': 'React resources',
+        'Getting_started_with_React': 'React 入门',
+        'Beginning_our_React_todo_list': '开始我们的 React todo 列表',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1233]:
```suggestion
        'Beginning_our_React_todo_list': '开始我们的 React 待办清单',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 3]
```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
...
+        'Componentizing_our_React_app': '组合我们的 React 应用',
+        'React_interactivity:_Events_and_state': 'React 交互：事件和状态',
+        'React_interactivity:_Editing_filtering_conditional_rendering': 'React 交互：编辑、筛选、条件渲染',
+        'Accessibility_in_React': 'React 无障碍',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1237]:
```suggestion
        'Accessibility_in_React': 'React 中的无障碍',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 4]
```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
...
-        'Componentizing_our_Svelte_app': 'Componentizing our Svelte app',
-        'Advanced_Svelte:_Reactivity_lifecycle_accessibility': 'Advanced Svelte: Reactivity, lifecycle, accessibility',
-        'Working_with_Svelte_stores': 'Working with Svelte stores',
-        'TypeScript_support_in_Svelte': 'TypeScript support in Svelte',
-        'Deployment_and_next_steps': 'Deployment and next steps',
+        'Getting_started_with_Svelte': 'Svelte 入门',
+        'Starting_our_Svelte_Todo_list_app': '开始我们的 Svelte Todo 列表应用',
+        'Dynamic_behavior_in_Svelte:_working_with_variables_and_props': 'Svelte 动态表现：变量和 props',
+        'Componentizing_our_Svelte_app': '组织我们的 Svelte 应用',
+        'Advanced_Svelte:_Reactivity_lifecycle_accessibility': '高级 Svelte: 响应式、生命周期和无障碍',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1261]:
```suggestion
        'Advanced_Svelte:_Reactivity_lifecycle_accessibility': '高级 Svelte：响应式、生命周期和无障碍',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 5]
```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
...
-        'Adding_a_new_todo_form:_Vue_events_methods_and_models': '添加一个新的TODO表单: Vue事件、方法、模型',
-        'Styling_Vue_components_with_CSS': '用CSS样式化Vue组件',
-        'Using_Vue_computed_properties': 'Using Vue computed properties',
-        'Vue_conditional_rendering:_editing_existing_todos': 'Vue conditional rendering: editing existing todos',
-        'Focus_management_with_Vue_refs': 'Focus management with Vue refs',
-        'Vue_resources': 'Vue resources',
+        'Getting_started_with_Vue': '开始使用 Vue',
+        'Creating_our_first_Vue_component': '创建第一个 Vue 组件',
+        'Rendering_a_list_of_Vue_components': '渲染一个 Vue 组件列表',
+        'Adding_a_new_todo_form:_Vue_events_methods_and_models': '添加一个新的 TODO 表单: Vue 事件、方法、模型',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1250]: 统一翻译成“待办清单”你看是不是更好些：

```suggestion
        'Adding_a_new_todo_form:_Vue_events_methods_and_models': '添加一个新的待办表单: Vue 事件、方法、模型',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 6]
```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
...
+        'Styling_Vue_components_with_CSS': '用 CSS 样式化 Vue 组件',
+        'Using_Vue_computed_properties': '使用 Vue 计算属性',
+        'Vue_conditional_rendering:_editing_existing_todos': 'Vue 条件渲染：编辑存在的 todo 列表',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1253]:
```suggestion
        'Vue_conditional_rendering:_editing_existing_todos': 'Vue 条件渲染：编辑存在的待办清单',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 7]
```diff
@@ -1131,143 +1131,144 @@ var text = mdn.localStringMap({
...
-        'Getting_started_with_Svelte': 'Getting started with Svelte',
-        'Starting_our_Svelte_Todo_list_app': 'Starting our Svelte Todo list app',
-        'Dynamic_behavior_in_Svelte:_working_with_variables_and_props': 'Dynamic behavior in Svelte: working with variables and props',
-        'Componentizing_our_Svelte_app': 'Componentizing our Svelte app',
-        'Advanced_Svelte:_Reactivity_lifecycle_accessibility': 'Advanced Svelte: Reactivity, lifecycle, accessibility',
-        'Working_with_Svelte_stores': 'Working with Svelte stores',
-        'TypeScript_support_in_Svelte': 'TypeScript support in Svelte',
-        'Deployment_and_next_steps': 'Deployment and next steps',
+        'Getting_started_with_Svelte': 'Svelte 入门',
+        'Starting_our_Svelte_Todo_list_app': '开始我们的 Svelte Todo 列表应用',
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:1258]:
```suggestion
        'Starting_our_Svelte_Todo_list_app': '开始我们的 Svelte 待办清单应用',
```

**Review by yin1999 — COMMENTED** [review]
some suggestions

**Comment by yin1999** [comment]
可以在 code review 中批量应用

File: kumascript/macros/LearnSidebar.ejs (hunk 1 above)
Comment (awxiaoxian2020) [reply kumascript/macros/LearnSidebar.ejs:1170]: @yin1999 We should update the context in some time.

**Comment by awxiaoxian2020** [comment]
It seems that re-request review not working for me. I click the button, no response.

**Comment by schalkneethling** [comment]
> It seems that re-request review not working for me. I click the button, no response.

Yeah, I believe only specific roles can re-request, unfortunately. Thank you for the PR and reviews, everyone!

**Review by schalkneethling — APPROVED** [review]
Thanks! 🎉

**Pull request merged**